app/src/main/java/com/ucm/informatica/spread/Activities/MainTabActivity.java

```diff
@@ -28,18 +43,22 @@
 
     private RelativeLayout relativeLayout;
 
+    private Map<Point, Region> regionMap = new HashMap<>();
+
+    private List<Event> dataSmartContractList = new ArrayList<>();
+
     private int[] tabIcons = {
```

Comment:
To decorate the upper tab, we need to order icons and names accordingly (probably would be better with a matrix  😶 )